parser/strconv.go

```diff
@@ -0,0 +1,34 @@
+package parser
```

Comment:
All Parse* methods in this package should return ast literal nodes. This remains consistent with all other Parse methods already in this package.

Reply:
Made the change, just need the thumbs up now.